app/serializers/recommendation_serializer.rb

```diff
@@ -2,7 +2,7 @@ class RecommendationSerializer
 
   def self.parse(playlist)
     playlist.reduce([]) do |acc, playlist|
-      acc << {id: playlist.id, name: playlist.name, url: playlist.url}
+      acc << {id: playlist.id, name: playlist.name, widget_url: playlist.url}
```

Comment:
Can we remove the widget url? Don't need this now. Thank you!

Reply:
Updated.

Reply:
URL was removed